sound/effect.h

```diff
@@ -465,6 +465,26 @@ EFFECT(ccbegin);
 EFFECT(ccend);
 EFFECT(ccchange);
 
+// Blaster effects
```

Comment:
A lot less overlap than I thought.
We really do need to figure out a way to only enable the effects that we need, but we don't need to do that in this CL.